src/core/psi_container.jl

```diff
@@ -499,14 +500,23 @@ function iterate_parameter_containers(psi_container::PSIContainer)
     end
 end
 
-function get_model_duals(op::PSIContainer, cons::Vector{Symbol})
-    results_dict = Dict{Symbol, DataFrames.DataFrame}()
-
-    for c in cons
-        v = get_constraint(op, c)
-        results_dict[c] = _result_dataframe_duals(v)
+function get_parameters_value(psi_container::PSIContainer)
+    # TODO: Still not obvious implementation since it needs to get the multipliers from
+    # the system
+    params_dict = Dict{Symbol, DataFrames.DataFrame}()
+    parameters = get_parameters(psi_container)
+    (isnothing(parameters) || isempty(parameters)) && return params_dict
+    for (k, v) in parameters
+        !isa(v.update_ref, UpdateRef{<:PSY.Component}) && continue
+        params_key_tuple = decode_symbol(k)
+        params_dict_key = Symbol(params_key_tuple[1], "_", params_key_tuple[3])
+        params_dict[params_dict_key] = axis_array_to_dataframe(get_parameter_array(v))
     end
-    return results_dict
+    return params_dict
+end
+
+function is_milp(container::PSIContainer)
```

Comment:
It looks like this is used to check if there are duals available. Should we generalize to any non LP?

Reply:
There is still value in the duals of constraints of NLP's. The issue is that this broke a lot of things in the export of results before we were exporting .feather files with a bunch of invalid values.